samples/QuickstartWeatherServer/Tools/WeatherTools.cs

```diff
@@ -0,0 +1,84 @@
+﻿using ModelContextProtocol.Server;
+using System.ComponentModel;
+using System.Net.Http.Headers;
+using System.Text.Json;
+
+namespace QuickstartWeatherServer.Tools;
+
+[McpServerToolType]
+public static class WeatherTools
+{
+    [McpServerTool, Description("Get weather alerts for a US state.")]
+    public static async Task<string> GetAlerts(
+        [Description("The US state to get alerts for.")] string state)
+    {
+        using HttpClient client = GetWeatherClient();
+
+        var response = await client.GetAsync($"/alerts/active/area/{state}");
+
+        if (!response.IsSuccessStatusCode)
+        {
+            return "Failed to retrieve alerts.";
+        }
+
+        var json = await response.Content.ReadAsStringAsync();
+        var jsonElement = JsonSerializer.Deserialize<JsonElement>(json);

```

Comment:
```suggestion
        var jsonElement = await client.GetFromJsonAsync<JsonElement>($"/alerts/active/area/{state}");
```

Reply:
I was originally going to create a class for the response, then I saw how complex it was and decided to just use `JsonElement` and never cleaned up the code.